src/python_service_template/api/coffee.py

```diff
@@ -0,0 +1,20 @@
+from typing import Annotated
+from fastapi import APIRouter, Depends, HTTPException
+from python_service_template.coffee.base import CoffeeDrink, CoffeeService
+from python_service_template.dependencies import coffee_service
+
+
```

Comment:
on the app structure - maybe we should consider using the same DDD-inspired structure that we follow in node-service-template, separating infrastructure from domain modules?

Does fastapi come with standard opinionated structure already?

Reply:
Hey, afaik there isn't any golden strandard in Python / FastAPI how to go about this but I hear you. I proposed something we might like more (I'm also fan of DDD project structure). Let me know your thoughts on this :)

Reply:
I love it! thank you